lib/tokenize.cpp

```diff
@@ -11376,12 +11376,26 @@ void Tokenizer::simplifyAsm()
         }
 
         else if (Token::Match(tok, "asm|__asm|__asm__ volatile|__volatile|__volatile__| (")) {
+            long balance = 0;
+            for (Token *uptok = tok; uptok && balance < 1; uptok = uptok->previous()) {
+               if (uptok->str() == "{")
```

Comment:
Is it possible to use `Token::link()` here?

Reply:
I suppose yes, and it'll work much faster.